theano/sandbox/cuda/cusolver.py

```diff
@@ -0,0 +1,159 @@
+from __future__ import absolute_import, division, print_function
+
+import pkg_resources
+import theano
+
+from theano.sandbox.cuda import CudaNdarray, GpuOp
```

Comment:
replace that by:

import theano.sandbox.cuda as cuda
from theano.sandbox.cuda import GpuOp

and use cuda.CudaNdarray elsewhere.

This will allow to import that file when cuda isn't available.